### PR TITLE
fix(DB/Creature):  Correct spells order for Broken-down Shredder (issue 18729)

### PR DESCRIPTION
**Pull request by xubpwg**
## Changes Proposed:

This PR proposes changes to:
-  [x] Database (creatures).

## Issues Addressed:

- Closes [18729](https://github.com/azerothcore/azerothcore-wotlk/issues/18729)

## SOURCE:

[Wrath retail](https://www.youtube.com/watch?v=7aVWto_rhGQ&t=47s)

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue

## Known Issues and TODO List:
[Issue comment](https://github.com/azerothcore/azerothcore-wotlk/issues/18729#issuecomment-2053769509) says that shredder can do melee attack on Classic, but I can't found any video for that behavior, so that question is still unclear

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/fix_issue18729.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+DELETE FROM `creature_template_spell` WHERE `CreatureID` = 27354;
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
+    (27354, 0, 48548, 12340),
+    (27354, 1, 48558, 12340),
+    (27354, 2, 48604, 12340),
+    (27354, 4, 48610, 12340);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/fix_issue18729.sql:7]:
```suggestion
INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
(27354, 0, 48548, 12340),
(27354, 1, 48558, 12340),
(27354, 2, 48604, 12340),
(27354, 4, 48610, 12340);
```

**Comment by Kitzunu** [comment]
> https://github.com/azerothcore/azerothcore-wotlk/issues/18729#issuecomment-2053769509 says that shredder can do melee attack on Classic, but I can't found any video for that behavior, so that question is still unclear

Bench is a reliable source, so if you can fix this it would be awesome

File: data/sql/updates/pending_db_world/fix_issue18729.sql (hunk 1 above)
Comment (heyitsbench) [reply data/sql/updates/pending_db_world/fix_issue18729.sql:7]: These entries need their `VerifiedBuild` set to 53788 as well. (Grabbed the first build I could find in which I have a sniff of this creature being charmed)

**Comment by elthehablo** [comment]
Thanks for the PR

**Review by elthehablo — APPROVED** [review]

**Pull request merged**